src/main/java/com/sogeti/SoSenseiPsyche/model/Code.java

```diff
@@ -0,0 +1,36 @@
+package com.sogeti.SoSenseiPsyche.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Code {
+    List<Character> code = new ArrayList<>();
+
+    private Color createRandomColour() {
+        int max = 8;
+        int randomNum = (int) (Math.random() * max);
+        return switch (randomNum) {
+            case 0 -> Color.BLUE;
+            case 1 -> Color.ORANGE;
+            case 2 -> Color.PINK;
+            case 3 -> Color.GREEN;
+            case 4 -> Color.RED;
+            case 5 -> Color.SILVER;
+            case 6 -> Color.WHITE;
+            case 7 -> Color.YELLOW;
+            default -> throw new IllegalStateException("Unexpected value: " + randomNum);
+        };
+    }
+
+    public void generateCode() {
+        for (int i = 0; i < 4; i++) {
+            code.add(createRandomColour().name().charAt(0));
+        }
+    }
+
+
+    public List<Character> getCode() {
+        generateCode();
+        return code;
+    }
```

Comment:
I know you did not create this but this could be put into one method no? We also discussed this